pkgs/development/libraries/libappindicator/default.nix

```diff
@@ -47,7 +47,6 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
```

Comment:
Is the rest of `postPatch` necessary with `autoreconfHook`?

Reply:
Yes you are right, fixed in a new commit